django/db/models/query.py

```diff
@@ -505,11 +505,12 @@ def _extract_model_params(self, defaults, **kwargs):
         params = {k: v for k, v in kwargs.items() if LOOKUP_SEP not in k}
         params.update(defaults)
         invalid_params = []
+        property_names = self.model._meta._property_names
```

Comment:
I think [the comment from `_property_names`](https://github.com/django/django/blob/6a3801e3ab76cf9a10ee171b1be4909743331371/django/db/models/options.py#L833) should change?